### PR TITLE
Rail Fence Cipher Implemented

### PR DESCRIPTION
**Pull request by prathameshpatil2002**

### CODE REVIEW

File: RailFenceEncrypter.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+#include <vector>
+#include "RailFenceEncrypter.h"
+
+using namespace std;
+
+void RailFenceCipherEncrypter ::initKey()
+{
+    // key = rand() % KEY_SIZE;
```
Comment (Harsh-Upadhayay) [inline RailFenceEncrypter.cpp]: Random key?

File: RailFenceEncrypter.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    for (auto it : msg)
+    {
+
+        if (!turn)
+        {
+            tmp[i++].push_back(it);
+            if (i == key)
+            {
+                i = key - 2;
```
Comment (Harsh-Upadhayay) [inline RailFenceEncrypter.cpp]: -2 ?

**Review by Harsh-Upadhayay — CHANGES_REQUESTED** [review]
Key initialization random kr Patil.

**Comment by prathameshpatil2002** [comment]
Kr diye changes

**Review by Harsh-Upadhayay — COMMENTED** [review]
merged

**Pull request merged**